.circleci/config.yml

```diff
@@ -97,31 +95,36 @@ jobs:
           path: reports/style
           destination: style
 
+  # Run unit tests for modified packages only with `--unit` flag
+  # Tested against Chrome and Firefox on Mac and Windows
   unit_tests:
-    <<: *job_common
+    executor: main-executor
     environment:
       COVERAGE: true
     steps:
-      - checkout
-      - <<: *restore_workspace
-      - <<: *restore_node_modules
-      - <<: *restore_packages
+      - checkout_and_fetch
+      - restore_node_modules
+      - restore_built_packages
+      - skip_check
```

Comment:
Since `skip_check` is not being utilized, let's leave it out until we decide we need it. I'd hate for someone to go through troubleshooting and spend time figuring out how this works.